### PR TITLE
[IMP] *: coherent hotkey uses

### PR DESCRIPTION
**Pull request by juliusc2066**
This commit makes hotkey uses more coherent throughout the entire codebase by setting alt+q as main shortcut for confirm and default actions and alt+x for cancel actions.

task-3370463

### CODE REVIEW

File: addons/purchase/views/purchase_views.xml [hunk 1]
```diff
@@ -140,7 +140,7 @@
                     <button name="action_create_invoice" string="Create Bill" type="object" class="oe_highlight" context="{'create_bill':True}" attrs="{'invisible': ['|', ('state', 'not in', ('purchase', 'done')), ('invoice_status', 'in', ('no', 'invoiced'))]}" data-hotkey="w"/>
                     <button name="action_rfq_send" states="sent" string="Re-Send by Email" type="object" context="{'send_rfq':True}" data-hotkey="g"/>
                     <button name="print_quotation" string="Print RFQ" type="object" states="sent" groups="base.group_user" data-hotkey="k"/>
-                    <button name="button_confirm" type="object" states="draft" context="{'validate_analytic': True}" string="Confirm Order" id="draft_confirm"/>
```
Comment (bouvyd) [inline addons/purchase/views/purchase_views.xml:143]: there is another one for the same method at line 138 which still uses `v`

**Review by bouvyd — COMMENTED** [review]
nitpicking i stustumbled upon

File: addons/account/wizard/account_payment_register_views.xml [hunk 2]
```diff
@@ -85,7 +85,7 @@
                     <footer>
                         <button string="Create Payments" name="action_create_payments" type="object" class="oe_highlight" data-hotkey="q" attrs="{'invisible': [('total_payments_amount', '=', 1)]}"/>
```
Comment (Polymorphe57) [inline addons/account/wizard/account_payment_register_views.xml:86]: no q?

File: addons/base_import_module/views/base_import_module_view.xml [hunk 3]
```diff
@@ -21,11 +21,11 @@
                     <footer>
                         <div states="init">
                             <button name="import_module" string="Import App" type="object" class="btn-primary"/>
```
Comment (Polymorphe57) [inline addons/base_import_module/views/base_import_module_view.xml:23]: add q?

File: addons/mass_mailing/wizard/mailing_contact_import_views.xml [hunk 4]
```diff
@@ -29,7 +29,7 @@
                     <button string="Import" type="object" name="action_import"
                         class="btn-primary" data-hotkey="i"/>
```
Comment (Polymorphe57) [inline addons/mass_mailing/wizard/mailing_contact_import_views.xml:30]: q?

File: addons/payment/wizards/payment_link_wizard_views.xml [hunk 5]
```diff
@@ -44,7 +44,7 @@
                     <button string="Close"
                             class="btn btn-secondary rounded-2"
                             special="cancel"
-                            data-hotkey="z"/>
```
Comment (Polymorphe57) [inline addons/payment/wizards/payment_link_wizard_views.xml:47]: no q above?

File: addons/stock/views/stock_move_views.xml [hunk 6]
```diff
@@ -183,15 +183,15 @@
                     <footer class="d-flex justify-content-between" attrs="{'invisible': ['|', ('state', '=', 'cancel'), '&amp;', ('state', '=', 'done'), ('is_locked', '=', True)]}">
                         <div class="d-flex gap-1">
                             <button string="Confirm" special="save" data-hotkey="v" class="oe_highlight"/>
```
Comment (Polymorphe57) [inline addons/stock/views/stock_move_views.xml:185]: q?

File: addons/stock_delivery/views/delivery_view.xml [hunk 7]
```diff
@@ -109,7 +109,7 @@
                         </div>
                     </group>
                     <footer>
-                        <button string="OK" special="cancel" data-hotkey="z" class="oe_highlight"/>
+                        <button string="OK" special="cancel" data-hotkey="x" class="oe_highlight"/>
```
Comment (Polymorphe57) [inline addons/stock_delivery/views/delivery_view.xml:112]: x or q?

File: odoo/addons/base/views/ir_profile_views.xml [hunk 8]
```diff
@@ -66,7 +66,7 @@
                     <field name="expiration"/>
                 </group>
                 <footer>
-                    <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z"/>
+                    <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="x"/>
                     <button string="Enable profiling" type="object" name="submit" class="btn btn-primary"/>
```
Comment (Polymorphe57) [inline odoo/addons/base/views/ir_profile_views.xml:70]: no q?

File: odoo/addons/base/wizard/base_language_install_views.xml [hunk 9]
```diff
@@ -37,7 +37,7 @@
                     </group>
                     <footer>
                         <button name="lang_install" string="Add" type="object" class="btn-primary"/>
```
Comment (Polymorphe57) [inline odoo/addons/base/wizard/base_language_install_views.xml:39]: no q?

File: odoo/addons/base/wizard/base_module_update_views.xml [hunk 10]
```diff
@@ -19,11 +19,11 @@
                     <footer>
                         <div states="init" class="d-flex gap-1">
                             <button name="update_module" string="Update" type="object" class="btn-primary"/>
```
Comment (Polymorphe57) [inline odoo/addons/base/wizard/base_module_update_views.xml:21]: no q?

**Comment by Polymorphe57** [comment]
robodoo r+

**Comment by bouvyd** [comment]
@alexkuhn replying to your discord DM here regarding the fact that CMD+Q will quit and using something close could be dangerous.

It's kinda true. The initial goal of choosing `q` all those year ago, I think, was to have the first letter of the keyboard as shortcut (on qwerty keyboards at least). It has survived ever since and is used in already many places, and I'd rather not change that now. Given the other shortcuts already in use in many places (e.g. p/n for navigation, h for home, c for create, s/j for save/discard, etc.) it's quite hazardous to try and change base shortcuts, so i'd rather not risk it `ㄟ( ▔, ▔ )ㄏ`

**Pull request closed without merging**